### ISSUE TITLE
Outdir: Save in timestamped subfolder?

### ISSUE THREAD

**Issue by joeflack4** [loinc/comp-loinc#86]
## Overview
The current behavior is that there is a configured default outdir, and within that, it saves the name of the recipe being run, and an optional inner subdir if running w/ fast-run. This setup isn't bad. I'm just wondering if we want to have a further `DATE` or `DATETIME` so that files don't get overwritten.

There is an advantage to files getting overwritten though; it reduces clutter.

Advantage to `DATE` over `DATETIME` is less clutter, and if it errors out, you can continue where you left off and it will put in the same folder.

I do like the idea of saving to a `DATETIME` subfolder.

**Comment by ShahimEssaid** [comment]
Yes, this would be useful. I had such an option in the previous implementation where outputs were placed in a timestamped subfolder of the output directory (to represent a specific run) and then also copied to the output directory. That way the output directory is the latest files of the various runs since not each run produced all the needed files. The timestamps on the files in the overall output directory helped tell which run produced them, and to know when an update of the other files might be needed.

**Comment by joeflack4** [comment]
I like that option as well.
